conduit/pschema/v1/grpc_client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/conduitio/conduit-connector-protocol/conduit/pschema"
+	"github.com/conduitio/conduit-connector-protocol/conduit/pschema/v1/fromproto"
+	"github.com/conduitio/conduit-connector-protocol/conduit/pschema/v1/toproto"
+	conduitv1 "github.com/conduitio/conduit-connector-protocol/proto/conduit/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var _ pschema.Service = (*Client)(nil)
+
+type Client struct {
+	grpcClient conduitv1.SchemaServiceClient
+}
+
+func NewClient() (*Client, error) {
+	conn, err := grpc.NewClient(
+		"localhost:8184",
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("failed creating gRPC client: %w", err)
+	}
+
+	return &Client{grpcClient: conduitv1.NewSchemaServiceClient(conn)}, nil
+}
+
+func (c *Client) Create(ctx context.Context, request pschema.CreateRequest) (pschema.CreateResponse, error) {
+	resp, err := c.grpcClient.Create(ctx, toproto.CreateSchemaRequest(request))
+	if err != nil {
+		return pschema.CreateResponse{}, fmt.Errorf("failed creating schema: %w", err)
```

Comment:
This will likely be in the error message already. That said, we should [unwrap the gRPC error](https://github.com/ConduitIO/conduit-connector-protocol/blob/c83ba94fe392faaaae56788f567b92692a10fe2e/pconnector/v2/client/errors.go#L37).